adhocteam/Head-Start-TTADP
# Program type constant updated. Only shows for grantee reports

**Description of change**

Program type constant updated to the values defined in the linked ticket. Program type field only shows when the report is for grantees, it is hidden for non-grantees.

**How to test**

1) Pull down
2) Start a report, select `grantee` and note `Program type(s)` is shown
3) Select `non-grantee` and note `Program type(s)` is hidden

**Issue(s)**
* https://github.com/HHS/Head-Start-TTADP/issues/170

**Checklist**

    - API methods
    - Boundary and Data Flow Diagrams
    - [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions with the [Nygard template](https://github.com/joelparkerhenderson/architecture_decision_record/blob/master/adr_template_by_michael_nygard.md)
    - OSCAL templates completed when security controls are implemented or modified
